datacapture/src/test/java/com/google/android/fhir/datacapture/extensions/MoreQuestionnaireItemComponentsTest.kt

```diff
@@ -1653,6 +1653,78 @@ class MoreQuestionnaireItemComponentsTest {
     assertThat(questionnaireItem.entryFormat).isNull()
   }
 
+  @Test
+  fun `createQuestionnaireResponseItem should set answer for non repeating question initial value`() {
+    val question =
+      Questionnaire.QuestionnaireItemComponent(
+          StringType("phone"),
+          Enumeration(
+            Questionnaire.QuestionnaireItemTypeEnumFactory(),
+            Questionnaire.QuestionnaireItemType.STRING
+          )
+        )
+        .apply {
+          initial = listOf(Questionnaire.QuestionnaireItemInitialComponent(StringType("000011111")))
+        }
+
+    val responseItem = question.createQuestionnaireResponseItem()
+
+    assertThat(responseItem.answer.map { it.value.primitiveValue() }).containsExactly("000011111")
+  }
+
+  @Test
+  fun `createQuestionnaireResponseItem should set answer for repeating question initial values`() {
+    val question =
+      Questionnaire.QuestionnaireItemComponent(
+          StringType("phones"),
+          Enumeration(
+            Questionnaire.QuestionnaireItemTypeEnumFactory(),
+            Questionnaire.QuestionnaireItemType.STRING
+          )
+        )
+        .apply {
+          repeats = true
+          initial =
+            listOf(
+              Questionnaire.QuestionnaireItemInitialComponent(StringType("000011111")),
+              Questionnaire.QuestionnaireItemInitialComponent(StringType("000022222"))
+            )
+        }
```

Comment:
optional : is it okay to update below change, can you please check at other places as well.

```suggestion
    val question =
      Questionnaire.QuestionnaireItemComponent()
        .apply {
          linkId = "phones"
          type = Questionnaire.QuestionnaireItemType.STRING
          repeats = true
          initial =
            listOf(
              Questionnaire.QuestionnaireItemInitialComponent(StringType("000011111")),
              Questionnaire.QuestionnaireItemInitialComponent(StringType("000022222"))
            )
        }
```